src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int NAME_INDEX = 0;
+    private static final int COUNT_POSITION = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = createReport(getDataFromFile(fromFileName));
+        writeDataToFile(toFileName, report);
+    }
+
+    private String getDataFromFile(String fromFile) {
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFile))) {
+            String line = reader.readLine();
+            while (line != null) {
+                builder.append(line).append(" ");
+                line = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file", e);
+        }
+        return builder.toString();
+    }
+
+    private String createReport(String data) {
+        String[] split1 = data.split(" ");
```

Comment:
```suggestion
        String[] records = data.split(" ");
```
names with numbers like this are not informative